CalculatorWithTests/Calculator.cs

```diff
@@ -1,8 +1,92 @@
-﻿
+﻿using System;
+
 namespace CalculatorWithTests
 {
-    class Calculator
+    public class Calculator
     {
-        
+        public int Addnums(int[] numbers)
+        {
+            int i = 0;
```

Comment:
This variable and the one in the foreach loop seem to have their names reversed. It looks like "i" is being used to keep the sum, and "sum" is each individual element of numbers. Consider swapping these names around